app/viewers/embed/viewer/geo.rb

```diff
@@ -67,7 +67,7 @@ def index_map_files
       end
 
       def default_height
-        493
+        new_viewer? ? super : 493
```

Comment:
I think `default_height` now needs to return a height string with units since `px` is no longer getting added in IframeComponent#height_style. It doesn't look like there are any tests that would have caught this for the geo viewer as there are for the file viewer.
```suggestion
        new_viewer? ? super : '493px'
```

Reply:
Except I think this will cause a problem for this comparison https://github.com/sul-dlss/sul-embed/pull/2653/files#diff-8f6c5a40f063a49b3d804a2dd5dd2bd828d879504714ed9ab9edb53204338ea9R20. I think you need to do 
```        return "#{default_height}px" if @embed_request.maxheight.to_i > default_height.to_i
```